contracts/interstake-yield-generator/src/msg.rs

```diff
@@ -38,9 +45,21 @@ pub enum ExecuteMsg {
     /// Claims rewards and then stake them; Only called by owner
     Restake {},
     /// Transfer amount of staked tokens to other address
-    Transfer { recipient: String, amount: Uint128 },
+    Transfer {
+        recipient: String,
+        amount: Uint128,
+        commission_address: Option<String>,
+    },
     /// Undelegates all tokens
     UndelegateAll {},
+    /// adds (or updates) address to allowed list
+    UpdateAllowedAddr {
+        address: String,
+        /// seconds since epoch
+        expires: Option<u64>,
```

Comment:
So the purpose is to add allowed addresses only for given period of time? I didn't get that from any comment or description.
Also, that's a surprising and arbitrary decision. Why?

Reply:
And on top of that if you choose None, it doesn't mean that there is no expiration but it's another arbitrary `MIN_EXPIRATION` period. 
I think `Option` should be removed, it will only surprise user later.

Reply:
Yea i kinda did it automatic. But i talked to kain and he likes the expiry.
 
Right. Ill change it to not optional.